consensus/src/execution_ctx.rs

```diff
@@ -115,7 +115,7 @@ impl<'a> ExecutionCtx<'a> {
                 // Timeout event. Phase could not reach its final goal.
                 // Increase timeout for next execution of this step and move on.
                 Err(_) => {
-                    tracing::info!(event = "timeout");
+                    tracing::info!(event = "timeout-ed");
```

Comment:
consider removing `tracing::`

Reply:
@moCello I removed all occurrences of `tracing::` usage